### PR TITLE
Bump SQLitePCLRaw from 2.1.0 to 2.1.1-pre20220822172036

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -13,6 +13,8 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <!-- TODO: Mirror packages and remove -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (bricelam) [inline NuGet.config:17]: Following up to mirror the closure:

- SQLitePCLRaw.bundle_e_sqlcipher
- SQLitePCLRaw.bundle_e_sqlite3
- SQLitePCLRaw.bundle_sqlite3
- SQLitePCLRaw.bundle_winsqlite3
- SQLitePCLRaw.core
- SQLitePCLRaw.lib.e_sqlcipher
- SQLitePCLRaw.lib.e_sqlite3
- SQLitePCLRaw.provider.dynamic_cdecl
- SQLitePCLRaw.provider.e_sqlcipher
- SQLitePCLRaw.provider.e_sqlite3
- SQLitePCLRaw.provider.sqlite3
- SQLitePCLRaw.provider.winsqlite3

**Comment by bricelam** [comment]
@smitpatel Can you review the query failures? Are they just more occurrences of #27129?

**Comment by smitpatel** [comment]
Looking into it

**Review by smitpatel — APPROVED** [review]

**Comment by smitpatel** [comment]
I had pushed a fix to skip the test, looks like it is gone in updating the branch. Broken build alert.

**Comment by bricelam** [comment]
Ugg. I bet I forgot to pull before squashing. I was wondering why "your" commit started with *fixup*. It didn't; that was one of mine.

**Comment by smitpatel** [comment]
Fixed it while merging rc1 to 7. so we are good again.

**Pull request merged**